Automattic/wp-calypso
# Titan: Fix incorrect name of Set Up Mailbox and Set Up Thank You pages

This pull request renames components, variables and urls of the `Set Up Mailbox` and `Set Up Thank You` pages to use the right word for the verb, i.e. `set up` instead of `setup`. For example, it renames `TitanSetupThankYou` to `TitanSetUpThankYou`, and changes https://wordpress.com/email/:domain/titan/setup-mailbox/:site to https://wordpress.com/email/:domain/titan/set-up-mailbox/:site. The logic didn't change, everything should work like before.

#### Testing instructions

1. Run `git checkout update/set-up-mailbox-page` and start your server, or access a [live branch](https://github.com/Automattic/wp-calypso/pull/55658#issuecomment-903775197)
2. Log into a WordPress.com account with a Professional Email account with unused mailboxes
3. Open the [`Emails` page](http://calypso.localhost:3000/email)
4. Click on that Professional Email account to access the `Email Plan` page
5. Click the `Set up mailbox` button
6. Assert that you see the right term in the url
7. Fill in the form, and click `Complete Setup`
8. Assert that you are presented with the `Thank You` page
9. Assert that you see the right term in the url
